docs/src/ref_impl/cross_compilation.md

```diff
@@ -5,15 +5,16 @@
 
 # Cross-Compilation
 
-Ghaf is targeted at a range of devices and form factors that support different instruction set architectures (ISA). Many small form-factor edge devices are not powerful enough to compile the needed applications or OSs that run on them. As the most common ISA used in desktops and servers is ``x\_86``, this will generally require that the code is cross-compiled for target ISA e.g. ``AArch64`` or ``RISC-V``.
+Ghaf is targeted at a range of devices and form factors that support different instruction set architectures (ISA). Many small form-factor edge devices are not powerful enough to compile the needed applications or OSs that run on them. As the most common ISA used in desktops and servers is ``x_86``, this will generally require that the code is cross-compiled for target ISA e.g. ``AArch64`` or ``RISC-V``.
 
 NixOS and Nixpkgs have good support for cross-compilation, however, there are still some that can not be compiled in this way.
 
```

Comment:
Cross-Compilation section is bit problematic if we are not open with Known Issues. I don't think it's currently possible to use Cross-Compilation on our supported targets.

Reply:
I just added to the very beginning "Cross-compilation is currently under development and cannot be used properly on our supported devices."
And that is one more reason why the Cross-Compilation section now is under the Development section.